server.js

```diff
@@ -77,8 +83,10 @@ app.get("/redemptions/:id", async(request, response) => {
   }
 });
 
-app.use(express.static("build"));
+if (process.env.NODE_ENV === "production") {
+  app.use(express.static("build"));
```

Comment:
in dev mode we've got 2 different servers:
* one is react-scripts - and it serves all the react stuff
* the other is the express api

so obviously for dev mode we don't want our express server to serve files from build folder - react-scripts server does that for us.